javaparser-core/src/main/java/com/github/javaparser/utils/CollectionStrategy.java

```diff
@@ -9,13 +9,17 @@
 import java.nio.file.PathMatcher;
 import java.util.Optional;
 
-import static com.github.javaparser.StaticJavaParser.parse;
+import static com.github.javaparser.StaticJavaParser.parse;import org.apache.log4j.Logger;
+
```

Comment:
Is this one of those `LexicalPreservingPrinter` issues?

Reply:
I don't know if there is a LexicalPreservingPrinter issue. The tool try to add an import with a default log adapter but there is an issue in the tool. I'm working on this.